### ISSUE TITLE
Does nativescript-exoplayer support livestream ?

### ISSUE THREAD

**Issue by ebizcoAU** [NathanaelA/nativescript-exoplayer#65]
Has anyone tried livestream with nativescript-exoplayer ? 
Any tip and demo with nativescript angular would be nice to share around
Google exoplayer supports livestream

**Comment by abdullahkhalidmahmood** [comment]
Yes, i have been trying myself however i am unable to find any solution yet.
